jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/resources/viewmodel/ResourceListViewModel.kt

```diff
@@ -19,6 +20,8 @@ class ResourceListViewModel : ViewModel() {
     var selectedGroupCardItem = SimpleObjectProperty<ResourceGroupCardItem>()
     val resourceGroupCardItemList: ResourceGroupCardItemList = ResourceGroupCardItemList()
 
+    val workbookProgressProperty = SimpleDoubleProperty(0.0)
```

Comment:
this might be confusing because it's not progress of the entire "workbook" but just the progress of this particular resource